client/src/main/java/com/arcbees/beestore/client/application/CurrentOrderImpl.java

```diff
@@ -30,11 +29,14 @@
 import com.google.web.bindery.event.shared.EventBus;
 
 public class CurrentOrderImpl implements CurrentOrder, HasHandlers {
+    private static final float TAX = 0.14975f;
```

Comment:
Also, we'd have to look what's going on if somebody from the US or another part of the world buy from us.